pkg/metrics/listener.go

```diff
@@ -29,24 +32,71 @@ var log = logf.RuntimeLog.WithName("metrics")
 // The metrics is on by default.
 var DefaultBindAddress = ":8080"
 
+// DefaultSecureBindAddress sets the default secure bind address for the metrics listener
+var DefaultSecureBindAddress = ":8843"
```

Comment:
I think we want to move the default metrics binding to `0`, ie disabled, so perhaps best to not introduce a default port for this, can we just reuse the other port for now?